setup.py

```diff
@@ -474,8 +68,7 @@ def chown_file(output):
         "License :: OSI Approved :: BSD License",
         "Operating System :: OS Independent",
         "Programming Language :: C",
-        "Programming Language :: Python :: 2.7",
-        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.6",
```

Comment:
```suggestion
        "Programming Language :: Python :: 3",
```

or do you require a specific minimum Python? In that case, list all compatible ones.